src/System.Memory/tests/Span/CopyTo.cs

```diff
@@ -48,7 +49,7 @@ public static void TryCopyToArraySegmentImplicit()
         [Fact]
         public static void TryCopyToEmpty()
         {
-            int[] src = {};
+            int[] src = { };
```

Comment:
Please note that I did't explicitly do this and other changes of whitespaces below... I'm not sure how they got introduced. But I think they are appropriate in terms of code formatting... If wanted, I will checkin another version that reinstates these changes...